tck/metrics/pom.xml

```diff
@@ -62,6 +66,7 @@
         <dependency>
             <groupId>io.opentelemetry</groupId>
             <artifactId>opentelemetry-semconv</artifactId>
+            <version>${version.otel.semconv}</version>
```

Comment:
As mentioned in #150 semantic conventions has new groupId and appears to "go back" in versions as they now track spec version instead of Java API version:

```
<dependency>
    <groupId>io.opentelemetry.semconv</groupId>
    <artifactId>opentelemetry-semconv</artifactId>
    <version>1.23.1-alpha</version>
</dependency>
```